e2e/js_image_oci/WORKSPACE

```diff
@@ -87,3 +87,38 @@ http_archive(
 load("@container_structure_test//:repositories.bzl", "container_structure_test_register_toolchain")
 
 container_structure_test_register_toolchain(name = "container_structure_test_toolchain")
+
+## Testing
```

Comment:
Isn't this entire WORKSPACE for testing?

Reply:
Yes, it is 👯  kind of pointless comment. will revert